bowling/inc/lane.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+class Lane {
+public:
+    Lane(const std::string& name) : name_(name) {}
+
+    size_t getPlayersNum() const { return players_.size(); }
+    std::string getPlayer(size_t index) { return players_.at(index); }
```

Comment:
could be `const` method

Reply:
validate index if not too high?

Reply:
added const, using at() with wrong index is save

Reply:
@m-kop  This code throws exception:
```
    Lane myLane{"ELO"};
    myLane.addPlayer("MN");
    myLane.addPlayer("XD");
    std::cout << myLane.getPlayer(0) << "\n";
    std::cout << myLane.getPlayer(1) << "\n";
    std::cout << myLane.getPlayer(2) << "\n";
```

Should we catch it?

Reply:
As we have problem with throwing exception, changed the code:)